lrzip/FeatureModel.xml

```diff
@@ -356,10 +160,25 @@
       <children />
       <impliedOptions />
       <excludedOptions />
-      <optional>True</optional>
-    </configurationOption>
-  </binaryOptions>
-  <numericOptions />
+      <optional>False</optional>
```

Comment:
FYI: numeric options currently can not be declared optional or not (i.e., numeric configuration options are always there). This is a flaw in the SPL Conqueror format.

Other than that, it looks good to me.

Reply:
Alright, I'll throw that out then